llvm-project/README.md

```diff
@@ -0,0 +1,67 @@
+# llvm-project
+
+## Goals
+To produce statically-linked builds of Clang and LLVM utilities supercharged
+for Linux kernel development, from a series of Docker images built
+continuously, that can be hosted on
+[kernel.org](https://mirrors.edge.kernel.org/pub/tools/crosstool/).
+
+### Stage N-4 goal (stage 1)
+- use llvm prebuilts from alpine to bootstrap
+- llvm-runtimes build (builds clang and lld, uses those to build runtimes)
+- no dependency on glibc
+
+### Stage N-3 goal (stage 2)
+- build clang and lld with llvm-runtimes from stage N-4
+- statically linked
+- no dependency on glibc
+- defaults to libc++, lld, compiler-rt, libunwind
+
+### Stage N-2 goal (stage 3)
+- statically linked
+- no dependency on glibc
+- no object files built from gcc
+- dependencies built from source
+
+### Stage N-1 goal (stage 4)
+- optional if PGO used instead of AutoFDO
+- collect profile data via PGO
```

Comment:
I am having a bit of a hard time parsing the first point here. I think it might be clearer to just explicitly state this stage is only done if we are doing PGO, maybe something like:

```
### Stage N-1 goal (stage 4)
- only built if doing PGO
- instrumented to collect profile data from building kernels
```

Reply:
done PTAL